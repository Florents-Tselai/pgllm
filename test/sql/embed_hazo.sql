select llm_embed('hello world', 'hazo');
