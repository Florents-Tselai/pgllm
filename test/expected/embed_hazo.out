select llm_embed('hello world', 'hazo');
             llm_embed             
-----------------------------------
 {5,5,0,0,0,0,0,0,0,0,0,0,0,0,0,0}
(1 row)

